modules/light-clients/07-tendermint/types/upgrade_test.go

```diff
@@ -481,3 +482,56 @@ func (suite *TendermintTestSuite) TestVerifyUpgrade() {
 		}
 	}
 }
+
+func (suite *TendermintTestSuite) TestUpdateStateOnMisbehaviour() {
+	var (
+		path *ibctesting.Path
+	)
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success",
+			func() {},
+			true,
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+
+		suite.Run(tc.name, func() {
+			// reset suite to create fresh application state
+			suite.SetupTest()
+			path = ibctesting.NewPath(suite.chainA, suite.chainB)
+
+			err := path.EndpointA.CreateClient()
+			suite.Require().NoError(err)
+
+			// ensure counterparty state is committed
+			suite.coordinator.CommitBlock(suite.chainB)
+
```

Comment:
I think this is unnecessary

Reply:
Indeed, copy pasta whoops! :D 
Removed!